src/main.ts

```diff
@@ -2,7 +2,7 @@ import authenticateDevice from './authentication.js';
 import notifyServerLoop from './scheduler.js';
 import { INITIAL_APP_STATE, INTERVAL_IN_MS } from './helpers/constants.js';
 
-global.applicationState = INITIAL_APP_STATE;
+globalThis.applicationState = INITIAL_APP_STATE;
```

Comment:
there's a chance this might break in node 10

Reply:
you are correct. esbuild does not handle this case.